Сhapter4/Chapter4.Task6/Program.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+//6. створити двовимірний масив, розміром n на m(n - кіслькість рядків, m - кількість стовпців). n та m не обовязково зчитувати з консолі.Заповнити довільними значеннями матрицю.Створити два одновимірні масиви. Перший, розміром n, кожен елемент якого - це сума чисел чисел у рядку. другий, розміром m, кожен елемент якого - це сума чисел чисел у стовпці
+
+namespace Chapter4.Task6
+{
+    class Program
+    {
+        static int ReadInt(string text)
+        {
+            Console.WriteLine(text);
+            string numStr = Console.ReadLine();
+            int num = int.Parse(numStr);
+
+            return num;
+        }
+        static void WriteMatrix(int[,] matrix, int numRows, int numColums)
+        {
+            for (int i = 0; i < numRows; i++)
+            {
+                for (int j = 0; j < numColums; j++)
+                {
+                    Console.Write("{0 } ", matrix[i, j]);
+                }
+                Console.WriteLine();
+            }
+        }
+        static void WriteArrays(int [] sumRowsArray, int [] sumColumnArray, int numRows, int numColumn)
```

Comment:
Method WriteArray should write only one array(any array) with any length
to write sumRowsArray and sumColumnsArray just call method 2 times
Do not pass array length into method. Think how to know array's length from array.